11ty/definitions/ok-hand.md

```diff
@@ -28,6 +28,6 @@ The uncertainty created by the 4chan-ers is a critical part of their strategy. T
 
 #### Impact
 
-The people most vulnerable to the oppressive impact of White Supremacy will never know for certain if the symbol is being used in one of its non-violent manners or if it's being used in its racist violent manners. The oppressed must choose the option that protects them, which is typically erring on the side of caution.
+The people most vulnerable to the oppressive impact of White Supremacy will never know for certain if the symbol is being used in a non-violent manner or if its being used in a racist violent manner. The oppressed must choose the option that protects them, which is typically erring on the side of caution.
```

Comment:
stet to both corrections here
- `it's` should stay it's here as is the contraction of `it is`
- white supremacy can be used in multiple violent manners, not just one. I worry that one implies less impact.

Reply:
understood!